src/collections/_documentation/performance/performance-metrics.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 'Performance Metrics'
+sidebar_order: 2
+---
+
+Sentry's application monitoring product collects performance metrics. Performance metrics provide a better understanding of how users are interacting with your application. By tracking metrics like Latency and Throughput in [Discover]({%- link _documentation/performance/discover/index.md -%}) or Performance, Sentry provides you with a fuller, real-time picture of your application's health, such as the occurrence of errors or performance issues.
+
+## Apdex
+Apdex is an industry-standard metric used to track and measure user satisfaction based on response times in your application. Your Apdex score is a measurement of the ratio of satisfactory, tolerable, and frustrated requests in a specific transaction or endpoint. This metric provides a standard for you to compare transaction performance, understand which ones may require additional optimization or investigation, and set targets or goals for performance.
+
+- Below are the various components of Apdex and its formula:
+    - T = Response time threshold for your target response time
+    - Satisfactory = T
+    - Tolerable = >T and ≤ 4*T
+    - Frustrated = 4*T
+    - Apdex = (# of Satisfactory Requests + (# of Tolerable Requests/2)) / (# of Total Requests)
```

Comment:
@MimiDumpling I think this could be revised in a better way.

- Let *T* be defined as the response time threshold for your target response time.
- Users are **satisfied** when using the app when their page load times are less than or equal to *T*.
- Users consider the app **tolerable** to use when their page load times are between *T* and *4T*.
- Users are **frustrated** with the app when their page load times are greater than *4T*.